vehicle/pacmod_interface/README.md

```diff
@@ -0,0 +1,15 @@
+# How to build and use AS node in Autoware
+
+1. Clone the autoware repository
+2. After cloning autoware get all the submodules.
+
+   ```sh
+   git submodule update --init --recursive
+   ```
+
+3. Build the workspace
+4. There is no GUI interface to launch this node. Hence use,
+
+   ```sh
+   roslaunch as ssc_interface.launch
```

Comment:
```suggestion
   roslaunch ssc_interface ssc_interface.launch
```

Reply:
README.md will be totally changed by the following PR, so let's not modify README.md here
https://github.com/tier4/AutowareArchitectureProposal.iv/pull/616

Reply:
OK!